src/include/processor/operator/ddl/add_rel_property.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "add_property.h"
+#include "storage/in_mem_storage_structure/in_mem_lists.h"
+
+using namespace kuzu::catalog;
+
+namespace kuzu {
+namespace processor {
+
+class AddRelProperty;
+
+using fill_in_mem_lists_function_t = std::function<void(AddRelProperty*, InMemLists* inMemLists,
+    uint8_t* defaultVal, PageByteCursor& pageByteCursor, node_offset_t nodeOffset,
+    list_header_t header, uint64_t posInList)>;
+using create_property_file_function_t = std::function<void(AddRelProperty*, table_id_t boundTableID,
```

Comment:
replaced RelStatistics with numTuples. RelsStatistics::getRelStatistics can't be removed, since i need this API to get a relStatistics which stores numTuples.